src/components/Auth/Auth.tsx

```diff
@@ -123,6 +124,15 @@ function Auth({
         </Container>
       )
       break
+    case VIEWS.UPDATE_PASSWORD:
+      return (
+        <Container>
+          <UpdatePassword
+            supabaseClient={supabaseClient}
+            setAuthView={setAuthView}
```

Comment:
The UpdatePassword component doesn't seem to have setAuthView prop

Reply:
@jin-exists you ok to fix this?

Reply:
The component does't use setAuthView at all so I'm not sure whether adding it is necessary?

Reply:
oh yea, sorry !

Reply:
I was asking to _remove_ the `setAuthView` prop btw, which you accidentally gave to the UpdatePassword component. 
I didn't make the proper changes I think so it looked like I've added it 😅

Reply:
Sorry everyone! I'm confusing everyone.
setAuthView could be removed, I can just make some changes on the develop branch :D